src/reducers/measurements/types.ts

```diff
@@ -0,0 +1,100 @@
+import { measurementIdSymbol } from "../../util/globals";
+
+// -- Measurements JSON types -- //
+
+/**
+ * Measurements are allowed to have arbitrary metadata.
+ * Matching types allowed in Augur's measurements schema
+ * <https://github.com/nextstrain/augur/blob/3f72c40897a80132099729d5d00a6718e76e0e9e/augur/data/schema-measurements.json#L152>
+ */
+type MeasurementMetadata = string | number | boolean
```

Comment:
Should this be renamed `JsonMeasurementMetadata` in 33afa42a00bc0c07219689ab93ddd6ce73e93063 to distinguish from `Measurement.[key: string]`?

Reply:
Hmm, yeah that makes sense!

Reply:
Fixed up in https://github.com/nextstrain/auspice/pull/1910/commits/4b165a45c9fe3a2fa10e3d9a4416eff991fad49e